ideascube/conf/idb_irq_bardarash.py

```diff
@@ -1,9 +1,17 @@
-# -*- coding: utf-8 -*-
-"""Bardarash in Kurdistan"""
+"""Bardarash in Kurdistan, Iraq"""
 from .azraq import *  # noqa
 from django.utils.translation import ugettext_lazy as _
 
-IDEASCUBE_NAME = u"Bardarash"
```

Comment:
Trivial nit: the commit message could have reflected this change. :stuck_out_tongue_winking_eye: